tony-core/src/main/java/com/linkedin/tony/Constants.java

```diff
@@ -15,6 +15,12 @@
   public static final String TASK_INDEX = "TASK_INDEX";
   public static final String TASK_NUM = "TASK_NUM";
   public static final String CLUSTER_SPEC = "CLUSTER_SPEC";
+
+  // PyTorch constants
```

Comment:
I think you should also add a comment on line 13 saying those are TensorFlow constants